code/modules/mob/living/simple_animal/hostile/mining/goliath.dm

```diff
@@ -30,13 +30,21 @@
 	move_force = MOVE_FORCE_VERY_STRONG
 	move_resist = MOVE_FORCE_VERY_STRONG
 	pull_force = MOVE_FORCE_VERY_STRONG
+	gender = NEUTER
+	sentience_type = SENTIENCE_OTHER
 	var/pre_attack = FALSE
 	var/pre_attack_icon = "Goliath_preattack"
 	loot = list(/obj/item/stack/sheet/animalhide/goliath_hide)
+	var/draconian = 0
```

Comment:
I would replace the numbers with something more telling, like NOT_DRACONIC for 0, etc. At the very least add a comment explaining possible values.

Reply:
Added a comment for the time being. It can either be 0, 1 or 2. The higher the number, the stronger the effect.